cmd/codegen/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+var buffer = bytes.NewBufferString("")
+
+func main() {
+	var specPath = flag.String("spec-path", "/home/masha/go/src/github.com/omc-college/management-system/api/rbac/rbac-api.yaml", "please use --spec-path /path/to/files/located")
```

Comment:
Change the default path to the absolute path and change the description to something like this: "path to rbac spec"